### PR TITLE
Fix/separation

### PR DESCRIPTION
**Pull request by QBYMKPI**
# Description

Terraform Module to create GCP Projects.

# How Has This Been Tested?

- [x] Apply & Destroy Terraform Code locally

# Checklist:

- [x] I have run tests and documented them above
- [x] I have performed a self-review of my own code
- [x] I have updated the documentation

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Please add headlines and descriptions for your usage examples. You can copy them from the google generated docs in README.

Please remove the google generated docs from README.

File: .terraform-docs.yml [hunk 1]
```diff
@@ -69,19 +87,25 @@ content: |-
   {{ include "examples/organization_policies/configs/boolean.yaml" }}
```
Comment (ghost) [inline .terraform-docs.yml]: add the name of the file on top of the code so the user knows what to do with it. `configs/boolean.yaml`

**Review by ghost — CHANGES_REQUESTED** [review]

File: .terraform-docs.yml (hunk 1 above)
Comment (QBYMKPI) [reply .terraform-docs.yml]: Put above hcl with description

File: .terraform-docs.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  {{- $filesResources := dict -}}
+  {{- $resourceTypes := dict -}}
+  {{- range .Module.Resources -}}
+    {{- template "setDict" dict "Dict" $filesResources "Key" .Position.Filename "Resource" . -}}
+    {{- $isResource := eq "resource" (printf "%s" .GetMode) -}}
+    {{- if $isResource -}}
+      {{- template "setDict" dict "Dict" $resourceTypes "Key" (printf "%s_%s" .ProviderName .Type) "Resource" . -}}
+    {{- end -}}
+  {{- end -}}
+  {{ indent 0 "#" }} Usage
```
Comment (ghost) [inline .terraform-docs.yml]:
```suggestion
```

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**